docs/13-walkthroughs/3-add-logging.md

```diff
@@ -0,0 +1,56 @@
+# Add Logging
+
+In this walkthrough, we are going to show how you can use a Zarf component to inject zero-config, centralized logging into your Zarf cluster.
+
+More specifically, you'll be adding a [Promtail / Loki / Grafana (PLG)](https://github.com/grafana/loki) stack to the [Doom Walkthrough](./2-deploying-doom.md) by installing Zarf's "logging" component.
+
+
+## Walkthrough Prerequisites
+1. The [Zarf](https://github.com/defenseunicorns/zarf) repository cloned: ([`git clone` Instructions](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository))
+1. Zarf binary installed on your $PATH: ([Install Instructions](../3-getting-started.md#installing-zarf))
+1. kubectl: ([kubectl Install Instructions](https://kubernetes.io/docs/tasks/tools/#kubectl))
```

Comment:
Think this is old, we shouldn't need kubectl I don't think...

Reply:
Removed, it is mentioned as a way to validate your cluster on the Zarf init walkthrough, but added a note to that to say it is optional.

Reply:
Yeah that was from before we did wait logic for cluster readiness or had k9s vendored in.

Reply:
Actually we should change this to "zarf tools monitor".  For our primary personas kubectl will have no advantage for k9s and in fact will be harder for many.

Reply:
yep that would definitely be better.  changed and added a small note on use.